examples/.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": [
+    "../.eslintrc"
+  ],
+
+  "globals": {
+    "pdfjsImageDecoders": false,
+    "pdfjsLib": false,
+    "pdfjsViewer": false,
+  },
+}
```

Comment:
Since the (various) builds are still transpiled to ES5 compatible code by default, the examples ought to be ES5 compatible as well. Hence please set the `"object-shorthand": "off",` rule here and revert the relevant changes in the next commit.